packages/viewer/src/modules/converter/Converter.js

```diff
@@ -173,6 +176,19 @@ export default class Coverter {
     this.activePromises -= childrenConversionPromisses.length
   }
 
+  directConverterExists(obj) {
+    return this[`${this.getSpeckleType(obj)}ToBufferGeometry`] !== undefined
+  }
+
+  getDisplayValue(obj) {
```

Comment:
So `displayMesh` was our really old version of fallback geometry values: I think the viewer ought to render only what's inside `displayValue` if it exists, and only to check for `displayMesh` if not (to maintain backwards compatibility)